src/controllers/controller.radar.js

```diff
@@ -133,14 +133,16 @@ module.exports = DatasetController.extend({
 			const options = me._resolveDataElementOptions(i);
 			const x = reset ? scale.xCenter : pointPosition.x;
 			const y = reset ? scale.yCenter : pointPosition.y;
+			const r = pointPosition.r;
 
 			// Utility
 			point._options = options;
 
 			// Desired view properties
 			point._model = {
-				x: x, // value not used in dataset scale, but we want a consistent API between scales
-				y: y,
+				x,
+				y,
+				r,
```

Comment:
It might be helpful to add some comments explaining the difference between `r`, `radius`, and `hitRadius` and why each is needed

Reply:
now that the `r` is renamed to `angle`, do you still need clarification between `radius` and `hitRadius`?